Sources/SpeziSpeechRecognizer/SpeziSpeechRecognizer.docc/SpeziSpeechRecognizer.md

```diff
@@ -0,0 +1,99 @@
+[your app in Xcode](https://developer.apple.com/documentation/xcode/adding-package-dependencies-to-your-app#) or
+[Swift package](https://developer.apple.com/documentation/xcode/creating-a-standalone-swift-package-with-xcode#Add-a-dependency-on-another-Swift-package).
+
+> Important: If your application is not yet configured to use Spezi, follow the [Spezi setup article](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/initial-setup) setup the core Spezi infrastructure.
+
+### 2. Configure the `SpeechRecognizer` in the Spezi `Configuration`
+
+The module needs to be registered in a Spezi-based application using the [`configuration`](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/speziappdelegate/configuration)
+in a [`SpeziAppDelegate`](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/speziappdelegate):
+```swift
+class ExampleAppDelegate: SpeziAppDelegate {
+    override var configuration: Configuration {
+        Configuration {
+            SpeechRecognizer()
+            // ...
+        }
+    }
+}
+```
+> Tip: You can learn more about a [`Module` in the Spezi documentation](https://swiftpackageindex.com/stanfordspezi/spezi/documentation/spezi/module).
+
+### 3. Configure target properties
+
+To ensure that your application has the necessary permissions for microphone access and speech recognition, follow the steps below to configure the target properties within your Xcode project:
+
+- Open your project settings in Xcode by selecting *PROJECT_NAME > TARGET_NAME > Info* tab.
+- You will need to add two entries to the `Custom iOS Target Properties` (so the `Info.plist` file) to provide descriptions for why your app requires these permissions:
+   - Add a key named `Privacy - Microphone Usage Description` and provide a string value that describes why your application needs access to the microphone. This description will be displayed to the user when the app first requests microphone access.
+   - Add another key named `Privacy - Speech Recognition Usage Description` with a string value that explains why your app requires the speech recognition capability. This will be presented to the user when the app first attempts to perform speech recognition.
+
+These entries are mandatory for apps that utilize microphone and speech recognition features. Failing to provide them will result in your app being unable to access these features. 
+
+## Example
+
+The code example demonstrates the usage of the Spezi ``SpeechRecognizer`` within a minimal SwiftUI application.
+
+```swift
+struct SpeechTestView: View {
+   // Get the `SpeechRecognizer` from the SwiftUI `Environment`.
+   @Environment(SpeechRecognizer.self) private var speechRecognizer
+   // The transcribed message from the user's voice input.
+   @State private var message = ""
+
+
+   var body: some View {
+        // Button used to start and stop recording by triggering the `microphoneButtonPressed()` function.
```

Comment:
Might be cool to show the message in the view, e.g. using a `Text("...")` view.

Same thing applies to the inline documentation.